tests/aviftest.c

```diff
@@ -317,16 +318,16 @@ static int runIOTests(const char * dataDir)
     static const char * ioSuffix = "/io/";
 
     char ioDir[FILENAME_MAX_LENGTH + 1];
-    int dataDirLen = (int)strlen(dataDir);
-    int ioSuffixLen = (int)strlen(ioSuffix);
+    size_t dataDirLen = strlen(dataDir);
+    size_t ioSuffixLen = strlen(ioSuffix);
 
     if ((dataDirLen + ioSuffixLen) > FILENAME_MAX_LENGTH) {
```

Comment:
Joe: If the reason you declared these variables as int was to avoid compiler warnings about signed-unsigned comparison here, I can change this test to:

`    if ((dataDirLen + ioSuffixLen) >= sizeof(ioDir)) {`